homeassistant/components/transmission/sensor.py

```diff
@@ -22,6 +28,76 @@
 )
 
 
+@dataclass
+class TransmissionSensorEntityDescription(SensorEntityDescription):
+    """Describe Transmission sensor entity."""
+
+    sub_type: str | None = None
+
+
```

Comment:
Can we set the values for the `key` same as the values of the `sub_type` and use it instead of `sub_type` below?